CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_zendesk v0.14.0
+
+[PR #136](https://github.com/fivetran/dbt_zendesk/pull/136) includes the following changes:
+
+## Bug Fixes
+- Converted the `zendesk__sla_policies` metric for `sla_elapsed_time` to be reported in minutes to the second as opposed to just the nearest rounded minute. This ensures more accurate reporting.
+- Updated the `int_zendesk__reply_time_combined` model to additionally account for the following business hour scenarios as they were erroneously being filtered out in previous versions of the package:
+   - A ticket is first replied to outside SLA schedules
+   - A ticket is has not yet received an agent reply
+- Included additional logic within the `int_zendesk__ticket_schedules` model to more accurately select the **active** default schedule.
+  - Previously the model could possibly select a deleted schedule. This update ensures only an active schedule is selected.
+- Overhauled the logic used to calculate `sla_breach_at` within the `zendesk__sla_policies` and upstream models for reply time SLAs. It was found this field was inconsistent with the actual breach/achieve time of an SLA. The overhaul should now ensure reply time SLA is accurate to either be the time of the SLA breach or achieve event.
+- Adjusted the `next_reply_time` SLA elapsed time metric calculation to also take into consideration the ticket solved event as a valid SLA event. Previously if a reply time SLA was attached to a ticket and there was no reply, but the ticket was closed then the SLA would be breached. This update ensures a closed event serves as a route for the SLA to be achieved or breached.
+- Modified the logic that matches schedule weeks when calculating reply time business metrics. Previously long running SLAs would be excluded from the final model, now all reply time business SLAs regardless of sla elapsed duration will be included in the end `zendesk__sla_policies` model.
+
+## Documentation Updates
+- Updated "Zendesk" references within the README to now refer to "Zendesk Support" in order to more accurately reflect the name of the Fivetran Zendesk Support Connector.
+- Added new entries to the DECISIONLOG to highlight nuances and opinionated stances this package uses when calculating business metrics and `first_reply_time` SLAs.
+
```

Comment:
don't forget to tag any relevant issues

Reply:
Do we usually tag the relevant issues? This PR includes the references to the Issues that will be addressed.

Reply:
Yeah and we typically will give the callout if the individuals contributed code or insight in addition to the original Issue. In this case there were no external contributions so we can keep exclude this. In the future we should still continue to highlight PRs and key issue contributors!